mpris2controller

```diff
@@ -207,6 +208,26 @@ class Controller(dbus.service.Object):
         log.info('Method call for Previous.')
         self.call_on_one_playing('Previous')
 
+    @dbus.service.method(dbus_interface=MY_INTERFACE)
+    def Status(self):
+        log.info('Method call for Status.')
+        status = ''
+        if self.playing:
+            status = Player(self.bus, list(self.playing)[0]).get('PlaybackStatus')
```

Comment:
`next(iter(self.playing))` seems to be the accepted way of peeking at an element in a set without needing an entirely new copy. So using that might be better.